stackoverflowex.py

```diff
@@ -0,0 +1,40 @@
+import pygame, random
```

Comment:
I am not sure what this file does. Header comment describing what this file does (and in other files) would be useful.